conf/base.config

```diff
@@ -10,32 +10,43 @@
  */
 
 process {
+  cpus   = { check_max( 1 , 'cpus'   ) }
+  memory = { check_max( 1.GB * task.attempt, 'memory' ) }
+  time   = { check_max( 4.h  * task.attempt, 'time'   ) }
 
-  container = params.container
-
-  // TODO nf-core: Check the defaults for all processes
-  cpus = { check_max( 1 * task.attempt, 'cpus' ) }
-  memory = { check_max( 8.GB * task.attempt, 'memory' ) }
-  time = { check_max( 2.h * task.attempt, 'time' ) }
-
-  errorStrategy = { task.exitStatus in [143,137] ? 'retry' : 'finish' }
-  maxRetries = 1
-  maxErrors = '-1'
+  errorStrategy = { task.exitStatus in [143,137,104,134,139] ? 'retry' : 'finish' }
+  maxRetries    = 1
+  maxErrors     = '-1'
 
   // Process-specific resource requirements
-  // TODO nf-core: Customise requirements for specific processes
-  withName: fastqc {
-    errorStrategy = { task.exitStatus in [143,137] ? 'retry' : 'ignore' }
-  }
-  withName: multiqc {
-    errorStrategy = { task.exitStatus in [143,137] ? 'retry' : 'ignore' }
+  // NOTE - Please try and re-use the labels below as much as possible.
+  //        These labels are used and recognised by default in DSL2 files hosted on nf-core/modules.
+  //        If possible, it would be nice to keep the same label naming convention when
+  //        adding in your local modules too.
+  // See https://www.nextflow.io/docs/latest/config.html#config-process-selectors
+  //withLabel:process_low {
```

Comment:
Here the statements `withLabel:process_low` `withLabel:process_medium` etc. should be uncommented, and you can use these labels to specify the resource requirements for each process in the pipeline.
Right now each of the processes is allowed only 1 CPU and 1.GB of memory (see the default values on lines 13-15).